Sprint-2/debug/2.js

```diff
@@ -1,8 +1,8 @@
 // Predict and explain first...
 
-const num = 103;
+// const num = 103; // deleted declaration of the num variable
```

Comment:
Why did you remove the declaration of `num` ?

Reply:
Because I wanted to make the num variable more flexible and assign a value when I call the function.